circle.yml

```diff
@@ -32,6 +34,7 @@ notify:
 test:
   override:
     - bundle exec rake lint:bash
+    - cd $GO_PROJECT && gometalinter.v1 --deadline=20s cmd/
```

Comment:
The rake task is supposed to run on the dev machine. On Circle the source code directory is in `./git-town`, so we have to `cd` into it.